reference/reference_test.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2020 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+package reference_test
```

Comment:
A blank line should separate a copyright header and the package directive, so that copyright header is not interpreted as package description by godoc.